docs/source/api/plugin/usage-reporting.md

```diff
@@ -323,5 +323,18 @@ Specify the function for creating a signature for a query. This option is not re
 </td>
 </tr>
 
+<tr>
+<td>
+
+###### `sendOperationDocumentsOnUnexecutableOperation`
+
+`Boolean`
+</td>
+<td>
+
+Whether to include the entire document in the trace if the operation was a GraphQL parse or validation error (i.e. failed the GraphQL parse or validation phases). This will be included as a separate field on the trace and the operation name and signature will always be reported with a cosntant identifier. Whether the operation was a parse failure or a validation failure will be embedded within the stats report key itself.
```

Comment:
cosntant -> constant

But also I think we can make this be a little more user-focused. Maybe something more along the lines of

> Statistics about operations that your server cannot execute (because the operation cannot be parsed or fails validation or has an operation name that doesn't match the operation body) are not reported separately to Apollo's servers, but are grouped together as "parse failure", "validation failure", or "unknown operation name". By default, the usage reporting plugin does not include the full operation document in reported traces, because it is challenging to strip potential private information (like string constants) from invalid operations. If you'd like the usage reporting plugin to include the full operation document so you can view it in Apollo Studio's trace view, set this to true.

The wording isn't ideal but I think this helps explain what's going on and why you might want to set it better?